app/controllers/merchant/dashboard_controller.rb

```diff
@@ -1,4 +1,9 @@
 class Merchant::DashboardController < Merchant::BaseController
   def index
+    user = User.find(session[:user_id])
+    @merchant = Merchant.find(user.merchant_id)
+    @orders = @merchant.orders
+    
+    require "pry"; binding.pry
```

Comment:
```@merchant.orders.joins(:item_orders)``` doesn't throw an error but nothing happens
```@merchant.orders.first.joins(:item_orders)``` does throw an error

Not sure what to do here since I don't want to iterate through ```@merchant.item_orders``` in the view, but don't know how to join each ```item_order``` price and quantity to ```@merchant.orders```